xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/AttachmentsPane.java

```diff
@@ -84,14 +105,19 @@ public void clickAttachFiles()
             .click();
     }
 
+    public int getAttachmentRowNb(String attachmentName)
+    {
+        return Integer.parseInt(getDriver()
```

Comment:
Wouldn't it make sense to have a generic method for that in LiveTableElement such as `getRowNumberForElement(FindBy findBy)` and use it here? Sounds like a common need.